bulletin-ui/src/components/Bulletin.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import BulletinHeader from './BulletinHeader';
+import BulletinMessage from './BulletinMessage';
+import AvatarIcon from './AvatarIcon';
+
+type BulletinProps = {
+  header: string;
+  ownerId: number;
+  message: string;
+};
+
+export default class Bulletin extends React.Component<BulletinProps> {
+  constructor(props: BulletinProps) {
```

Comment:
Not needed constructor. I can also see warnings in console 
![image](https://user-images.githubusercontent.com/12229968/62939643-65cd8a00-bdda-11e9-87c4-d3ebdb7968b9.png)